src/skipwhile.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * Skipwhile emitted values from source until provided expression is false.
+ */
+export function skipWhile<T>(predicate: (t: T) => boolean = () => true) {
+  return async function* inner(source: AsyncIterable<T>) {
+    for await (const item of source) {
+      if (!predicate(item)) {
```

Comment:
Ah, I don't think this will do the right thing. It should skip while the predicate is false and than return everything after that. Something like:

```javascript
let skipping = true
if(skipping) {
  if(predicate(item)) {
    skipping = false;
    yield item;
  }
} else {
  yield item;
}

Also, could you add docs for the functions in docs/API.md? Thanks!

Reply:
@jamiemccrindle  can you give me a source where my implementation will fail ?

Reply:
@kedar9444 see the comment in the test case